packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
```

Comment:
1) You could  use destructuring assignment as well:
```
getLabels({tags}){
  const tags = test.tags   <<<<----- can remove
```
2) If you prefer to put object rename it from `test` to `suite`

Reply:
This is a mistake. i'm fixing that.